### ISSUE TITLE
disposeamail.com working?

### ISSUE THREAD

**Issue by hoasis2** [vlucas/vancelucas-site#2]
Dear Vance
Is the website disposeamail.com working right now?
I have not received any mails for a few weeks or longer.
Regards

**Comment by vlucas** [comment]
Yes, I have noticed. It seems to be a DNS issue with Namecheap. No MX records are being returned from Namecheap, even though they are configured and have not been edited in years. I am moving all the DNS config over to Cloudflare and am hopeful it will fix the issues.

**Comment by hoasis2** [comment]
Thank you for taking care of this. 
Aren't the problems [somewhere else](https://mxtoolbox.com/emailhealth/disposeamail.com/)?

```
smtp 	mx2.disposeamail.com 	Failed To Connect
smtp 	mx1.disposeamail.com 	Failed To Connect 

```

**Comment by hoasis2** [comment]
Dear Vance
Would you please fix this page soon? Any help would be appreciated. :-)
Best regards
hoasis2

**Comment by hoasis2** [comment]
I still hope that this is possible... ;-)

Unfortunately, I have to recover an account.

Happy new year!
Cheers

**Comment by hoasis2** [comment]
In the meantime, I wonder why this website is still online even though it hasn't worked for months? Scams cannot be ruled out here. 👎 

You should be on your guard!
